project/styles/components/content/collapsibles.scss

```
@@ -94,7 +94,7 @@
 
     @include breakpoint($bp-l) {
       top: 30px;
-      right: 21px;
+      left: 21px;
```

Ik zou hier left: 18px doen, ik vind het erg dicht op de tekst staan. 
```suggestion
      left: 18px;
```

---

Met Stijn overlegd en het was ok.